tests/schema-tests/package.json

```diff
@@ -15,9 +15,5 @@
     "generate-schema:orchestration": "pnpm generate-config:orchestration && pnpm ts-to-zod --all --skipValidation",
     "generate-schema": "pnpm generate-schema:azure-openai && pnpm generate-schema:orchestration",
     "postgenerate-schema": "rm ./ts-to-zod.config.cjs"
-  },
-  "devDependencies": {
-    "ts-to-zod": "^3.15.0",
-    "zod": "^3.24.2"
```

Comment:
[q] Was removing this causing any issue to you (maybe in the pipeline)? If I remember correctly, I made a compromise here to not let the workflow fail. But maybe it works now idk.

Reply:
Nope, pnpm should pick the dependency from the root node_modules.